scripts/build.sh

```diff
@@ -52,6 +52,9 @@ main() {
   local plugins="-Dbundle_plugins="
   local prefix=/
   local build_type="release"
+  local sdl3_version="${SDL3_VERSION:-3.2.8}"
```

Comment:
We could already update to `3.2.10`.

Reply:
sorry the comment was lost, but 3.2.10 has a build issue with HIDAPI. You'll have to dig into commit history to find it.

Reply:
https://github.com/libsdl-org/SDL/commit/98e76d283fd164005d07ab5e95bc01b398a7ab91